app/controllers/admin/stores_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::StoresController < ApplicationController
+before_action :require_admin
+  def index
+    @active_stores = Store.active_stores
+  end
+
+  def pending_stores
+    @pending_stores = Store.pending_stores
+    render :pending
+  end
+
+  def suspended_stores
+    @suspended_stores = Store.suspended_stores
+    render :suspended
+  end
+
```

Comment:
Nice use of scopes here. I probably would've setup my controller differently to try to stick to the 'basic' crud actions.  Instead of creating endpoints for pending stores & suspended stores I would've used params and an if statement in the index action.